tmdb/episode.go

```diff
@@ -32,8 +32,8 @@ func GetEpisode(showID int, seasonNumber int, episodeNumber int, language string
 		Params: napping.Params{
 			"api_key":                apiKey,
 			"append_to_response":     "credits,images,videos,alternative_titles,translations,external_ids,trailers",
-			"include_image_language": fmt.Sprintf("%s,en,null", config.Get().Language),
-			"include_video_language": fmt.Sprintf("%s,en,null", config.Get().Language),
+			"include_image_language": fmt.Sprintf("%s,%s,null", config.Get().Language, config.Get().SecondLanguage),
```

Comment:
If your main language is `en`, then `SecondLanguage == ''`. 
Had to test how they will return images with `en,,null`, but it looks fine.

Reply:
i also tested this, as mentioned in description:
>include_image_language=en,,null